test/test_nestedtensor.py

```diff
@@ -3033,6 +3033,17 @@ def test_device_dtype_transfer_maintains_offsets(self, device, dtype):
             self.assertEqual(orig_device, nt.offsets().device)
             self.assertEqual(torch.int64, nt.offsets().dtype)
 
+    def test_unbind(self, device):
```

Comment:
thanks for adding this :) there's def some gaps in op test coverage for the subclass because it's so easy to support ops. at some point, we should look into generic testing (possibly involving OpInfos) to fill these gaps